src/loading/regular_folder_accessor.cpp

```diff
@@ -32,12 +32,11 @@ namespace nova::renderer {
         std::fseek(resource_file, 0, SEEK_END);
         const auto file_size = std::ftell(resource_file);
 
-        std::string file_string;
-        file_string.resize(file_size);
+        std::string file_string(file_size, 0);
 
         std::fseek(resource_file, 0, SEEK_SET);
 
-        std::fread(file_string.data(), sizeof(char), file_size, resource_file);
+        std::fread(&file_string[0], sizeof(char), file_size, resource_file);
```

Comment:
Why?

Reply:
Because someone on GP suggested that `.data()` isn't supposed to let you modify the contents of the string

Reply:
hwat